src/hooks/screen/useChatScreenHook.ts

```diff
@@ -152,6 +191,12 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     sendChat,
     handleUserName,
     updateChatContinuity,
+    flatListRef,
+    scrollContext: {
+      selectedMessageId,
+      setSelectedMessageId,
+      handleScrollTo
+    },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return object of `useChatScreenHook` now includes `flatListRef` and `scrollContext`. This seems to be related to the new scrolling feature in the chat. Ensure these are used appropriately in the code.
```diff
+ flatListRef,
+ scrollContext: {
+   selectedMessageId,
+   setSelectedMessageId,
+   handleScrollTo
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->